tests/unit/awaitStagingDeploysTest.ts

```diff
@@ -40,16 +59,18 @@ const mockListWorkflowRuns = jest.fn().mockImplementation((args) => {
 
 beforeAll(() => {
     // Mock core module
-    core.getInput = mockGetInput;
+    asMutable(core).getInput = mockGetInput;
 
     // Mock octokit module
-    const moctokit = {
+    const moctokit: InternalOctokit = {
         rest: {
+            // @ts-expect-error This error was removed because getting the rest of the data from internalOctokit makes the test to break
```

Comment:
Why is this needed?

Reply:
This is needed because InternalOctoKit requires all the data of the object to be passed but in this case for the unit tests we shall only pass listWorkflowRuns otherwise it will break the test